### PR TITLE
Added functionality to open new tabs right after current tab.

### LINKED ISSUE #636: Open new tab just right after the current one
When you open (right clic) a link in a new tab, would be better to have it just after the current active one.

First reported in #635

### PR DESCRIPTION
**Pull request by juuz0**
Fixes #636

### CODE REVIEW

File: src/tabbar.cpp [hunk 1]
```diff
@@ -127,10 +127,15 @@ void TabBar::setNewTabButton()
     setTabButton(idx, QTabBar::RightSide, Q_NULLPTR);
 }
 
-ZimView* TabBar::createNewTab(bool setCurrent)
+ZimView* TabBar::createNewTab(bool setCurrent, bool adjacentToPreviousTab)
```
Comment (mgautierfr) [inline src/tabbar.cpp:130]: It may be better to name the argument `adjacentToCurrentTab`

**Review by mgautierfr — CHANGES_REQUESTED** [review]
Just a small change in the parameter name and we can merge.

**Comment by juuz0** [comment]
Should I squash?

**Comment by mgautierfr** [comment]
> Should I squash?

Yes please. The two commits are not two intermediate steps. The second is a fix for the first one. So no need to keep both changes separated (but it was nice to have the two commits to simplify the review)

**Comment by juuz0** [comment]
Alright, done.

**Review by mgautierfr — APPROVED** [review]

**Comment by mgautierfr** [comment]
I've rebased and fix the conflict.

**Pull request merged**